cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,143 +3,31 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
-.org-chart {
 
-    .respond-to-min(@bp-sm-min, {
-        width: 90%;
-        margin: 0 auto;
-    });
+@grid_gutter-width__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@spine-width__px: 5px;
+@spine-color: #BABBBD;
```

Comment:
This is @gray-50 which is no longer on our color pallette. Should we update to `@gray-40`? @ajbush @duelj?